osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -190,15 +190,15 @@ public void Add(DrawableHitObject fruit)
             /// <returns>Whether the catch is possible.</returns>
             public bool AttemptCatch(CatchHitObject fruit)
             {
-                const double relative_catcher_width = CATCHER_SIZE / 2;
+                const double relative_catcher_width = CATCHER_SIZE;

```

Comment:
this is probably correct, but should be in its own comit or own PR

Reply:
I transported the `/2` statement to `validCatch` variable to unite it